oteapi/strategies/parse/image.py

```diff
@@ -93,53 +143,85 @@ def get(
         self, session: "Optional[Dict[str, Any]]" = None
     ) -> SessionUpdateImageParse:
         """Execute the strategy."""
-        if session:
-            self._use_filters(session)
-        session = session if session else {}
+        if not session:
+            session = {}
+
+        config = self.parse_config.configuration
+        crop = config.crop if config.crop else session.get("imagecrop")
 
         mime_format = self.parse_config.mediaType.split("/")[1]
-        image_format = self._map_mime_to_format()
+        image_format = SupportedFormat[mime_format].value
+
+        # Proper download configurations
+        conf = self.parse_config.dict()
+        conf["configuration"] = config.download_config
+        download_config = ResourceConfig(**conf)
```

Comment:
```suggestion
        download_config = self.parse_config.copy()
        download_config.configuration = config.download_config
```

Reply:
If this doesn't work, you could use `**download_config` in the `create_strategy()` calls below.